assets/samples/dynakube/v1beta3/logMonitoring.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: dynatrace.com/v1beta3
+kind: DynaKube
+metadata:
+  name: dynakube
+  namespace: dynatrace
+spec:
+  apiUrl: https://ENVIRONMENTID.live.dynatrace.com/api
+
+  # Link to api reference for further information: https://docs.dynatrace.com/docs/ingest-from/setup-on-k8s/reference/dynakube-parameters
+
+  # For more context to logMonitoring please refer to: https://docs.dynatrace.com/docs/ingest-from/setup-on-k8s/reference/dynakube-parameters
```

Comment:
this feels too similar to the link above, both link to the same api reference.

We can align the wording to what is used in the official documentation, there should be a decision by now.